src/components/configurable-items/configurable-items.scss

```diff
@@ -0,0 +1,16 @@
+.carbon-configurable-items {
+  .carbon-form__buttons {
+    position: relative;
+  }
+
+  .carbon-form__additional-actions {
```

Comment:
I've added a PR to add support for this https://github.com/Sage/carbon/pull/1429